radicle-cli/src/commands/patch.rs

```diff
@@ -83,7 +91,9 @@ pub enum Operation {
     Checkout {
         patch_id: Rev,
     },
-    List,
+    List {
+        filter: Option<patch::State>,
```

Comment:
Maybe `filter_state`, `with_state`, or even `state`?  Something which describes the attribute value used for filtering.

Reply:
`Option<State>` is the filter. `with_state` and `filter_state` wouldn't make sense when set to `None`. `state` wouldn't communicate the intent, and we already know it's the state by looking at the type.

Reply:
> we already know it's the state by looking at the type

Using the type to infer its use is adding a level of indirection and we shouldn't expect user interfaces for the source code to do it.

Reply:
Not sure what you mean.. my argument boils down to `state` being less informative than `filter`.